doc/classes/ArrayOccluder3D.xml

```diff
@@ -24,10 +24,12 @@
 		<member name="indices" type="PackedInt32Array" setter="set_indices" getter="get_indices" default="PackedInt32Array()">
 			The occluder's index position. Indices determine which points from the [member vertices] array should be drawn, and in which order.
 			[b]Note:[/b] The occluder is always updated after setting this value. If creating occluders procedurally, consider using [method set_arrays] instead to avoid updating the occluder twice when it's created.
+			[b]Note:[/b] The returned value is a copy of the original. Methods which mutate the size or properties of the return value will not impact the original array. To change properties of the array, assign it to a temporary variable and make changes before reassigning the class property.
```

Comment:
One more thing that contributes to the issue. "_Properties_ of the array" in this context can potentially be ambiguous. What it is referring to is the PackedInt32Array of `indices` (specifically its elements), but `indices` **is** a _property_ itself.